api_inference_community/validation.py

```diff
@@ -175,6 +175,7 @@ class StringInput(BaseModel):
     "conversational": ConversationalInputsCheck,
     "question-answering": QuestionInputsCheck,
     "feature-extraction": StringOrStringBatchInputCheck,
+    "language-identification": StringInput,
```

Comment:
Is there any reason to add a task ? It seems this is **exactly** text-classification, no ?

Reply:
Ah, you're right. I added it because it was failing tests but I mistakenly assigned "language-identification" to `TASKS` like:

```python
os.environ["TASK"] = "language-identification"
```

[Changed it](https://github.com/huggingface/api-inference-community/blob/94171fdb935522f2b2e4f198404d758cf67e137c/docker_images/fasttext/tests/test_api_language_identification.py#L20) to just use text-classification.